<?xml version="1.0"?>
<launch>

	<node pkg="airo_control" type="airo_control_node" name="airo_control_node" output="screen">
        <rosparam command="load" file="$(find airo_control)/config/vicon_param.yaml" />
	</node>

</launch>
